polars/polars-core/src/series/series_trait.rs

```diff
@@ -509,7 +509,7 @@ pub trait SeriesTrait:
         polars_bail!(opq = is_in, self._dtype());
     }
     #[cfg(feature = "repeat_by")]
-    fn repeat_by(&self, _by: &IdxCa) -> ListChunked {
+    fn repeat_by(&self, _by: &IdxCa) -> PolarsResult<ListChunked> {
         invalid_operation_panic!(repeat_by, self)
```

Comment:
Now that we return a Result here, we should replace the invalid_operation with:

```rust
polars_bail!(opq = repeat_by, self._dtype());
```

Reply:
updated